src/app/modules/shared_models/section_item.nim

```diff
@@ -51,7 +51,7 @@ type
     historyArchiveSupportEnabled: bool
     pinMessageAllMembersEnabled: bool
     bannedMembersModel: member_model.Model
-    pendingMemberRequestsModel: member_model.Model
+    pendingMemberRequestsModel*: member_model.Model
```

Comment:
I don't see the usage of it from the outside, is * needed?

Reply:
SectionModel accesses it